src/server/scripts/Spells/spell_paladin.cpp

```diff
@@ -184,29 +216,29 @@ class spell_pal_blessing_of_faith : public SpellScript
 
     void HandleDummy(SpellEffIndex /*effIndex*/)
     {
-        if (Unit* unitTarget = GetHitUnit())
```

Comment:
indentation, also unrelated change i guess

Reply:
yup, not sure why it even changed; don't remember touching that. Thanks